verde/utils.py

```diff
@@ -207,6 +209,116 @@ def maxabs(*args, nan=True):
     return npmax(absolute)
 
 
+def build_grid(
+    coordinates,
+    data,
+    data_name,
+    dims=("northing", "easting"),
+    extra_coords_name=None,
+    attrs=None,
+):
+    """
+    Convert gridded data to :class:`xarray.Dataset`
+
+    Parameters
+    ----------
+    coordinates : tuple of arrays
+        Arrays with coordinates of each point in the grid. Each array must
+        contain values for a dimension in the order: easting, northing,
+        vertical, etc. All arrays need to have the same *shape*. These
+        coordinates can be generated through :func:`verde.grid_coordinates`.
+    data : array or tuple of arrays
+        Array or tuple of arrays with data values on each point in the grid.
+        Each array must contain values for a dimension in the same order as
+        the coordinates. All arrays need to have the same *shape*.
+    data_name : str or list
+        String or list containing strings with the names of the data arrays.
+    dims : list or None
+        The names of the northing and easting data dimensions, respectively,
+        in the output grid. Must be defined in the following order: northing
+        dimension, easting dimension.
+        **NOTE: This is an exception to the "easting" then
+        "northing" pattern but is required for compatibility with xarray.**
+    extra_coords_name : str or list
+        Name or list of names for extra coordinates. Ignored if coordinates has
+        only two elements.
```

Comment:
Need to be very clear about what are dims and how they are different from coordinates and extra_coords_name.